### PR TITLE
Expose identity verifications

### PR DESCRIPTION
**Pull request by erikdstock**
This pr adds an identity verification query by id under the `me` type. I copied the date type's implementation from another file and removed some l8n logic from what i copied - see a4e21e0. Otherwise I think this is good to go. Thanks to @sweir27 for the pair on testing.

### CODE REVIEW

File: src/schema/v2/me/identity_verification.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+const IdentityVerificationType = new GraphQLObjectType<any, ResolverContext>({
+  name: "IdentityVerificationType",
+  fields: {
+    ...InternalIDFields,
+    state: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "Where the identity verification is in its lifecycle",
+    },
+    userID: {
+      type: new GraphQLNonNull(GraphQLString), // TODO: Ok to have an id string(?) here instead of the User object?
```
Comment (erikdstock) [inline src/schema/v2/me/identity_verification.ts]: Minor question here.

File: _schemaV2.graphql [hunk 2]
```diff
@@ -4701,6 +4701,24 @@ type HomePageRelatedArtistArtworkModule {
   basedOn: Artist
 }
 
+type IdentityVerificationType {
```
Comment (mzikherman) [inline _schemaV2.graphql]: I think the `...Type` suffix for this type is redundant (when seeing it in the SDL form here). Can probably drop that from the name.

File: src/schema/v2/me/identity_verification.ts (hunk 1 above)
Comment (mzikherman) [reply src/schema/v2/me/identity_verification.ts]: It's up to you, and what the schema needs are. I would probably leave out entirely unless you actually need it.

Comment (ashkan18) [reply src/schema/v2/me/identity_verification.ts]: i _think_ its ok to have it as ID for now till there is a reason to resolve it with actual user. Knowing this is already under `me` i don't think we need that just yet.

File: src/lib/loaders/loaders_with_authentication/gravity.ts [hunk 3]
```diff
@@ -6,6 +6,13 @@ export type StartIdentityVerificationGravityOutput = {
   identity_verification_flow_url: string
 }
 
+export type IdentityVerificationGravityResponse = {
```
Comment (mzikherman) [inline src/lib/loaders/loaders_with_authentication/gravity.ts]: Wondering why this (and the other type above), are exported from this file - are they being used anywhere? This is a more general file for all authenticated loaders, feels a bit out of place.

#minor

**Comment by erikdstock** [comment]
I'm going to go ahead and merge this - to be clear I spoke with @ashkan18 about the `invitation_expires_at` field and we are on the same page (it is already correct).

File: src/schema/v2/me/identity_verification.ts [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+> = {
+  ...dateField,
+  resolve: (
+    { invitation_expires_at: expiresAt },
+    { format, timezone },
+    { defaultTimezone }
+  ) => {
+    const rawDate = expiresAt
+
+    const timezoneString = timezone ? timezone : defaultTimezone
```
Comment (damassi) [inline src/schema/v2/me/identity_verification.ts]: `timezone || defaultTimezone`

**Comment by erikdstock** [comment]
Thanks all, responded to all feedback and going to #mergeongreen

**Pull request merged**